binding/SkiaSharp.Skottie/Animation.cs

```diff
@@ -53,17 +79,20 @@ public unsafe void Render (SKCanvas canvas, SKRect dst)
 		public void Render (SKCanvas canvas, SKRect dst, AnimationRenderFlags flags)
 			=> SkottieApi.skottie_animation_render_with_flags (Handle, canvas.Handle, &dst, flags);
 
-		public void Seek (double t, InvalidationController ic = null)
-			=> SkottieApi.skottie_animation_seek (Handle, (float)t, ic?.Handle ?? IntPtr.Zero);
+		public void Seek (double percent, InvalidationController? ic = null)
+			=> SkottieApi.skottie_animation_seek (Handle, (float)percent, ic?.Handle ?? IntPtr.Zero);
 
-		public void SeekFrame (double t, InvalidationController ic = null)
-			=> SkottieApi.skottie_animation_seek_frame (Handle, (float)t, ic?.Handle ?? IntPtr.Zero);
+		public void SeekFrame (double frame, InvalidationController? ic = null)
+			=> SkottieApi.skottie_animation_seek_frame (Handle, (float)frame, ic?.Handle ?? IntPtr.Zero);
 
-		public void SeekFrameTime (double t, InvalidationController ic = null)
-			=> SkottieApi.skottie_animation_seek_frame_time (Handle, (float)t, ic?.Handle ?? IntPtr.Zero);
+		public void SeekFrameTime (double seconds, InvalidationController? ic = null)
+			=> SkottieApi.skottie_animation_seek_frame_time (Handle, (float)seconds, ic?.Handle ?? IntPtr.Zero);
 
-		public double Duration
-			=> SkottieApi.skottie_animation_get_duration (Handle);
+		public void SeekFrameTime (TimeSpan time, InvalidationController? ic = null)
+			=> SeekFrameTime (time.TotalSeconds, ic);
+
+		public TimeSpan Duration
+			=> TimeSpan.FromSeconds(SkottieApi.skottie_animation_get_duration (Handle));
```

Comment:
Added the new `SeekFrameTime(TimeSpan)` and changed `Duration` to be a `TimeSpan`.